### PR TITLE
Major update of latest template from nf-core

### PR DESCRIPTION
**Pull request by ypriverol**
## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/quantms/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/quantms _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by jpfeuffer** [comment]
I like it better if we have one PR for template merge. And one PR for version update.

**Comment by jpfeuffer** [comment]
but template update is already merged.

**Comment by jpfeuffer** [comment]
please pull from upstream

**Comment by jpfeuffer** [comment]
All tests fail.
We need to fix:
- conda
- proteininference in TMT. Did no one adapt FDRs or something?
- dumpsoftwareversions not working anymore
- multiqc with DIA not working

**Comment by ypriverol** [comment]
We will take care of all the issues @daichengxin @WangHong007

**Comment by ypriverol** [comment]
> All tests fail. We need to fix:
> 
> * conda
I will take care of this one 
> * proteininference in TMT. Did no one adapt FDRs or something?
I guess this is a more complex issue @jpfeuffer. Did we change anything in thw TMT algorithm? 
> * dumpsoftwareversions not working anymore
@daichengxin can you take carw of this ☝️
> * multiqc with DIA not working
@WangHong007 can you check why DIA is not working

**Comment by jpfeuffer** [comment]
I think I saw that the test was failing before the update already.

**Comment by WangHong007** [comment]
@ypriverol @jpfeuffer In all DIA test, the identification and quantification work was skipped, and Dia-NN did not work, which caused pmultiqc not to receive the diann_report and mistook it for the DDA type.

**Comment by ypriverol** [comment]
We should run the identification and quant steps.

**Comment by jpfeuffer** [comment]
@ypriverol can you maybe do a new PR where you are just changing the version numbers. a) this will have a much cleaner commit history and b) we can be more sure about errors caused by this

**Comment by ypriverol** [comment]
@jpfeuffer @WangHong007 @daichengxin are working already in the PR.

**Comment by jpfeuffer** [comment]
Ok. I just think you would save some time. Because most of the errors are not present on nf-core.

**Comment by jpfeuffer** [comment]
So I think the reason is the very complicated merge in this PR.

**Comment by daichengxin** [comment]
Hi @jpfeuffer, All issues are solved except for TMT. I tested TMT locally, it also reported same problems. But I tested it using openms 2.8.0, it's ok.  These are two proteininference input files from openms 2.8.0 and  2.9.0. Could you review? Thanks a lot.
2.9.0
[ID_mapper_merge.zip](https://github.com/bigbio/quantms/files/10920427/ID_mapper_merge.zip)
2.8.0
[ID_mapper_merge.zip](https://github.com/bigbio/quantms/files/10920541/ID_mapper_merge.zip)

**Comment by jpfeuffer** [comment]
We probably need the idXMLs that were used in IDMapper.

**Comment by daichengxin** [comment]
[TMT_Erwinia_1uLSike_Top10HCD_isol2_45stepped_60min_01-20141210_.zip](https://github.com/bigbio/quantms/files/10920892/TMT_Erwinia_1uLSike_Top10HCD_isol2_45stepped_60min_01-20141210_.zip)

**Comment by timosachsenberg** [comment]
thanks I will look into it immediately

**Comment by timosachsenberg** [comment]
fixed in https://github.com/OpenMS/OpenMS/pull/6758

File: CHANGELOG.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ The pipeline is using Nextflow DSL2, each process will be run with its own [Bioc
 | `thermorawfileparser` | 1.3.4      |
 | `comet`               | 2021010    |
 | `msgf+`               | 2022.01.07 |
-| `openms`              | 2.8.0      |
+| `openms`              | 2.9.0      |
```
Comment (jpfeuffer) [inline CHANGELOG.md:63]: 291

**Review by jpfeuffer — CHANGES_REQUESTED** [review]
Pyopenms was not updated since bioconda is bugged.

**Comment by ypriverol** [comment]
@jpfeuffer we are ready. You can merge the PR.

**Review by jpfeuffer — APPROVED** [review]
I hope merging is gonna be fine since it's a messy PR that somehow still shows the template changes. But other than that, looks good.

**Pull request merged**